lib/qesapdeployment.pm

```diff
@@ -91,6 +102,95 @@ sub qesap_get_deployment_code {
         push(@log_files, QESAP_GIT_CLONE_LOG);
         assert_script_run("set -o pipefail ; $git_clone_cmd | tee " . QESAP_GIT_CLONE_LOG, quiet => 1);
     }
+    assert_script_run("ln -s " . TERRAFORM_DIR . "/aws " . TERRAFORM_DIR . "/ec2");
```

Comment:
This is a problem with terraform directory naming in qe-sap-deployment project and what is expected by PC terraform library. 
In qesap-deployment project uses 'aws' but openqa library works with 'ec2'.
The least intrusive way for both to be working is a symlink. 

Google is fine since terraform directories are the same. If you use 'PUBLIC_CLOUD_PROVIDER': GCP there is no need for a symlink.